.github/workflows/final.yml

```diff
@@ -1,24 +1,15 @@
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+  pull_request_review:
+    types:
+      - submitted
   release:
     types:
       - published
-name: build-test
+
+name: complete-conformance-check
```

Comment:
I don't know if conformance is the right word, as that sounds too close to the dist spec conformance tests.

Reply:
And my previous comment still stands, the person who opened the reviw will have to wait for someone else to review before we run the final check. Can we do it using labels instead?